src/pyxplor/plot_time_series.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
I dont think you need to import matplotlib and np here cuz they just appear in the docstring.

Reply:
```suggestion
```

Reply:
I can remove that part, I don't think it matter!